lib/ProMotion/table/cell/table_view_cell_module.rb

```diff
@@ -63,10 +63,10 @@ def set_remote_image
       self.imageView.image = remote_placeholder
 
       if sd_web_image?
-        @remote_image_operation = SDWebImageManager.sharedManager.downloadWithURL(data_cell[:remote_image][:url].to_url,
-          options:SDWebImageRefreshCached,
+        @remote_image_operation = SDWebImageManager.sharedManager.loadImageWithURL(data_cell[:remote_image][:url].to_url,
```

Comment:
This change will break for anyone who is still using v3. You could change this to support both v3 and v4 by checking to see if the method exists before calling it, like this:
```
if SDWebImageManager.sharedManager.respond_to?('downloadWithURL:options:progress:completed')
  # old v3 code here
else
  # new v4 code here
end
```